Modules/Core/Common/include/itkBresenhamLine.hxx

```diff
@@ -61,6 +40,12 @@ BresenhamLine<VDimension>::BuildLine(LType Direction, IdentifierType length) ->
   {
     LastIndex[i] = (IndexValueType)(length * Direction[i]);
   }
+
+  // Direction of increment. -1 or 1
+  IndexType m_OverflowIncrement;
+  // Increment for the error for each step. Two times the difference between
+  // start and end
+  IndexType m_IncrementError;
```

Comment:
This commit seems unrelated to the commit message "One declaration per line for readability".